Replay/Replay/Assets.xcassets/App Icon & Top Shelf Image.brandassets/App Icon - Large.imagestack/Middle.imagestacklayer/Content.imageset/Contents.json

```diff
@@ -4,6 +4,10 @@
       "idiom" : "tv",
```

Comment:
Changes to this project file is not required. Lets make sure it doesn't show up.